docs/src/design/story/wireframes/v1_3/v12-styles.css

```
@@ -0,0 +1,1267 @@
+.spec-card .spec-body ol li,
+.spec-card .spec-body ul li {
+  margin: 0 0 10px 0;
+  padding-left: 2px;
+}
+.spec-card .spec-body ol li:last-child,
+.spec-card .spec-body ul li:last-child {
+  margin-bottom: 0;
+}
+.spec-card .spec-body ol li code,
+.spec-card .spec-body ul li code {
+  word-break: break-word;
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)